### DISCUSSION TITLE
Import settings from Sequel Pro

### DISCUSSION THREAD

**Discussion by patpawlowski** [Sequel-Ace/Sequel-Ace#634] [General] [answered]
HI all, thanks for keeping this codebase going. Is there a way to import all my connections from Sequel Pro? 

Thanks, 

-pat

**Comment by Jason-Morcos** [comment, marked as answer]
There's some info on our website on how to migrate! :)
https://sequel-ace.com/get-started/migrating-from-sequel-pro.html
